rust/automerge/src/exid.rs

```diff
@@ -68,6 +69,31 @@ impl ExId {
         }
     }
 
+    #[inline(never)]
+    pub fn to_unsafe_u8(&self) -> Vec<u8> {
```

Comment:
I guess this is necessary for performance? If so then given that these are public functions we should document what they do and _why_ they are unsafe.